style.css

```diff
@@ -0,0 +1,268 @@
+.body {
+   margin: 0;
+ }
+
+ .header {
+   display: flex;
+   align-items: center;
+ }
+
+ .header::before {
+   content: '';
+   position: absolute;
+   width: 461px;
+   height: 1800px;
+   background: #000;
+ }
+
+ .header__title {
+   position: relative;
+   width: 98px;
+   height: 37px;
+   margin: 36px 0 0 52px;
+   font-family: Montserrat;
+   font-style: normal;
+   font-weight: 500;
+   font-size: 30px;
+   line-height: 37px;
+   text-align: center;
+   letter-spacing: .07em;
+   color: #fff;
+ }
+
+ .header__language {
+   position: relative;
+   display: flex;
+   width: 238px;
+   justify-content: flex-end;
+   align-items: center;
+   top: 15px;
+ }
+
+ .header__language-ru {
+   position: relative;
+   width: 31px;
+   height: 25px;
+   font-family: Montserrat;
+   font-style: normal;
+   font-weight: 500;
+   font-size: 20px;
+   line-height: 24px;
+   color: #FFF;
```

Comment:
Codacy found an issue: [Expected "#FFF" to be "#fff" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Locc/pullRequest?prid=5986341)